torch/_torch_docs.py

```diff
@@ -2845,6 +2845,36 @@ def merge_dicts(*dicts):
         tensor([False,  True,  False,  True,  False])
 """)
 
+add_docstr(torch.isposinf,
+           r"""
+isposinf(input, out=None) -> Tensor
+Returns a new tensor with boolean elements representing if each element is `+INF` or not.
+Args:
+    {input}
+    {out}
```

Comment:
Keyword args:
  {out}

Reply:
`{input}` is removed.

Reply:
See later comment on how to style this. Sorry about the confusion.